
The new copy_user stuff return value does yield warnings and thus we can not
compile the sparc dirs with -Werror ...

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.10/arch/sparc64/kernel/Makefile.vanilla	2005-02-01 20:35:16.000000000 +0100
+++ linux-2.6.10/arch/sparc64/kernel/Makefile	2005-02-01 20:35:30.000000000 +0100
@@ -3,7 +3,7 @@
 #
 
 EXTRA_AFLAGS := -ansi
-EXTRA_CFLAGS := -Werror
+EXTRA_CFLAGS := 
 
 extra-y		:= head.o init_task.o vmlinux.lds
 
--- linux-2.6.10/arch/sparc64/lib/Makefile.vanilla	2005-02-01 20:35:53.000000000 +0100
+++ linux-2.6.10/arch/sparc64/lib/Makefile	2005-02-01 20:35:58.000000000 +0100
@@ -3,7 +3,7 @@
 #
 
 EXTRA_AFLAGS := -ansi
-EXTRA_CFLAGS := -Werror
+EXTRA_CFLAGS := 
 
 lib-y := PeeCeeI.o copy_page.o clear_page.o strlen.o strncmp.o \
 	 memscan.o strncpy_from_user.o strlen_user.o memcmp.o checksum.o \
--- linux-2.6.10/arch/sparc64/mm/Makefile.vanilla	2005-02-01 20:35:39.000000000 +0100
+++ linux-2.6.10/arch/sparc64/mm/Makefile	2005-02-01 20:35:47.000000000 +0100
@@ -3,7 +3,7 @@
 #
 
 EXTRA_AFLAGS := -ansi
-EXTRA_CFLAGS := -Werror
+EXTRA_CFLAGS := 
 
 obj-y    := ultra.o tlb.o fault.o init.o generic.o extable.o
 
